docs/contributing.md

```diff
@@ -0,0 +1,61 @@
+- Make sure all Azure changes work with a [Private VNet deployment](#setting-up-a-private-vnet). We're currently working on including this in integration testing so all verification must happen manually.
+
+## Developer Environment
+### Getting Started
+Instructions for getting your environment set up and building the executable can be found [here](https://github.com/microsoft/CromwellOnAzure#optional-build-the-executable-yourself). Once your build successfully deploys the Azure resources and runs a test workflow you're good to go. 
+
+If you run into any problems or want more information and ways to customize deployments and runs, please see the [Troubleshooting Guide](https://github.com/microsoft/CromwellOnAzure/blob/main/docs/troubleshooting-guide.md). 
+### So you're ready to contribute?
+You've read the above section on Issues and Pull Requests and you're ready to jump in! We ask that your contributions follow our coding standards and your changes work with a private vnet deployment.
+
+#### Coding Standards 
+Cromwell on Azure currently does not have enforced linting, so we do ask all contributors to help keep things neat. We generally follow the [C# Style Conventions](https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions) (though not as much on camel casing private variables). 
+
+This includes things like 
+- Keeping one newline between end braces `}` 
+- Using `var` instead of explicitly typed variables
+- Keeping our `using` statements in alphabetical order
+- Cleaning up unused variables
+- Making full use of modern C# conventions such as omitting type name or using object initalizer for the [`new` keyword](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/new-operator#constructor-invocation) and using modern operators such as [is](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/is) (ex: `vnetAndSubnet is not null` instead of `vnetAndSubnet != null`)
+- Using [Asynchronous Programming](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/async/) to effectively allocate resources and block as little as possible.
+
+We also ask contributors to plaese update the docs if you add a flag to Configuration.cs.
+
+#### Setting up a Private VNet
+TODO: REPLACE THIS SECTION WITH LINK TO PRIVATE VNET DOC ONCE JONATHON PUSHES IT.
+
+Deploying Cromwell on Azure in a Private Virtual Network ensures a secure self contained deployment with no public endpoints. This deployment expects you to have already created a number of your own resources hooked up to the VNet. 
+
+Azure resources to manually set up (must all be deployed in same region): 
+- Virtual Network (with Managed PostgreSQL subnet if applicable) and Network Security Group
+- A VM in the virtual network to use at a bridge machine. (You won't be able to run Cromwell on Azure from your local machine with the Private VNet.)
```

Comment:
i think you mean "as a bridge machine"

Reply:
Thank you for proofreading, no spell check/grammar check in VSCode :'( 
Changes made!